package.json

```diff
@@ -9,10 +9,18 @@
   "devDependencies": {
     "@builder.io/qwik": "~1.3.2",
     "@builder.io/qwik-city": "~1.3.2",
+    "@nestjs/common": "^10.0.2",
+    "@nestjs/core": "^10.0.2",
+    "@nestjs/platform-express": "^10.0.2",
```

Comment:
would be great if you migrate to Fastify by default, for the performance reason.

Reply:
I have migrate from express to fastify, please help check changes again.